airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -204,7 +204,11 @@ function getDefaultCursorField(streamNode: SyncSchemaStream): string[] {
   return streamNode.config.cursorField;
 }
 
-const useInitialSchema = (schema: SyncSchema): SyncSchema =>
+const useInitialSchema = (
+  schema: SyncSchema,
+  destDefinition: DestinationDefinitionSpecification,
+  isEditMode?: boolean
+): SyncSchema =>
```

Comment:
Can you please add some unit tests for this hook. Since we have a well defined requirement for how this should behave and what it can return, we should have unit tests covering this behavior.